config/console.php

```diff
@@ -9,16 +9,16 @@
 return [
     Extractor::class => [
         '__construct()' => [
-            // Please set the following to use extractor
+            // Please set the following to use extractor. MessageReader and MessageWriter should be set to SAME MessageSource (One folder in PHP MessageSource)
             'messageReader' => static function () {
                 throw new \RuntimeException('You should configure MessageReader.');
             },
             'messageWriter' => static function () {
-                throw new \RuntimeException('You should  configure MessageWriter.');
+                throw new \RuntimeException('You should configure MessageWriter.');
```

Comment:
Because keep the exception, you can set it by default.

Reply:
And if the user uses DB Source?

Reply:
Well that's a point, you could have a commented configuration .

Reply:
I'm removed commented lines from config. Examples of config - in readme.md